doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -90,7 +90,7 @@ see [`mz_cluster_replica_utilization`](#mz_cluster_replica_utilization).
 
 ### `mz_cluster_replica_utilization`
 
-The `mz_cluster_replica_utilization` table gives the last known CPU and RAM utilization statistics
+The `mz_cluster_replica_utilization` view gives the last known CPU and RAM utilization statistics
```

Comment:
Wait, I thought this didn't make the release.

Reply:
It didn't, but the documentation for `mz_cluster_replica_sizes` _and_ `mz_cluster_replica_utilization` was bundled up in the implementation PR of (#16597). I can default to removing these cases when putting together the notes and opening up a PR that can be merged when the actual feature is released.